src/App.jsx

```diff
@@ -19,6 +19,24 @@ class App extends React.Component {
   }
 
   render() {
+    var lock = new Auth0Lock('KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH', 'loom.auth0.com');
+    var btn_login = document.getElementById('btn-login');
```

Comment:
this is not the React way to do things. it's probably ok for now to get things going, but it's worth investing a little bit of time to do this the right way